### ISSUE TITLE
esbuild-plugin-copy Could not resolve dependency

### ISSUE THREAD

**Issue by kaykhan** [LinbuduLab/nx-plugins#88]
I get the following error when i try to install this package

```
npm install esbuild-plugin-copy --save-dev 
```

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: email-service@1.0.0
npm ERR! Found: esbuild@0.15.10
npm ERR! node_modules/esbuild
npm ERR!   dev esbuild@"^0.15.10" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer esbuild@"^0.14.0" from esbuild-plugin-copy@1.3.0
npm ERR! node_modules/esbuild-plugin-copy
npm ERR!   dev esbuild-plugin-copy@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/kay/.npm/eresolve-report.txt for a full report.

```

**Comment by linbudu599** [comment]
I'll update peer deps for copy-plugin in recently, for now you can install with `--force` / `--legacy-peer-deps`(but I'm not sure does the plugin api still works correctly at 0.15.0), or you can just install @^0.14.0 for the project.

**Comment by linbudu599** [comment]
fixed in 1.5.1

**Comment by xrzhuang** [comment]
@linbudu599 I have this same issue for the `node-externals` plugin
